Microsoft.Toolkit.Uwp.UI.Controls/InfiniteCanvas/InfiniteCanvas.xaml

```diff
@@ -75,7 +75,7 @@
                                         <Button.Flyout>
                                             <Flyout Placement="Bottom">
                                                 <ColorPicker x:Name="CanvasTextBoxColorPicker"
-                                                             Color="Red" />
```

Comment:
When had a problem here, unfortunately when you set the color picker to black and then open the color picker it gives a very black view and you had to change the slider on the left to get a color then be able to choose a color which is not easy. that's why we made it Red :)